temporal-object/src/main/java/com/iluwatar/temporalobject/CreditContractVersion.java

```diff
@@ -36,9 +37,20 @@ public class CreditContractVersion {
   private final int cvcCode;
   private final SimpleDate expirationDate;
 
-  CreditContractVersion(String contractText, String contractIssuer,
-                        String version, int creditLimit, int cardNumber, int cvcCode,
-                        SimpleDate expirationDate) {
+  /**
+   * Creator for a credit card contract version.
+   *
+   * @param contractText The text contained in the contract.
+   * @param contractIssuer The issuer of the contract.
+   * @param version The version of the contract.
+   * @param creditLimit The credit limit according to the contract.
+   * @param cardNumber The credit number associated to the contract.
+   * @param cvcCode The cvc code associated to the contract.
+   * @param expirationDate The date the contract's associated card is no longer valid.
+   */
```

Comment:
I don't think you need a Javadoc like this on a constructor. Sorry if I suggested so in my previous review.

Reply:
This docstring was requested by the Google Checks check style, though I think for a large constructer such as this it's probably best to have it.

Reply:
Okay cool. Thank you for clarifying that.